samples/zigbee/network_coordinator/README.rst

```diff
@@ -16,17 +16,19 @@ It is a minimal implementation that supports only the network steering commissio
 Requirements
 ************
 
-* One of the following development kits:
+The sample supports the following development kits:
 
-  * |nRF52840DK|
-  * |nRF52833DK|
+.. include:: /includes/boardname_tables/sample_boardnames.txt
+   :start-after: set8_start
+   :end-before: set8_end
 
-* One or both of the following samples:
+You can use one or more of the development kits listed above and mix different development kits.
 
-  * The :ref:`Zigbee light bulb <zigbee_light_bulb_sample>` sample programmed on one or more separate devices.
-  * The :ref:`Zigbee light switch <zigbee_light_switch_sample>` sample programmed on one or more separate devices.
+For this sample to work, one or both of the following samples need to be programmed:
```

Comment:
The following samples (light bulb, light switch) are not mandatory. They are optional,  **Testing** section explains that these samples are optional and testing steps that involve them can be skipped